ModelBase/src/util/NameResolver.cpp

```diff
@@ -40,7 +40,7 @@ QList<QPair<QString, Node*>> NameResolver::mostLikelyMatches(const QString& node
 	auto matcher = SymbolMatcher(new QRegExp(pattern, Qt::CaseInsensitive, QRegExp::Wildcard));
 
 	for (auto manager : AllTreeManagers::instance().loadedManagers())
-		matches.append(findAllMatches(matcher, "", manager->root()));
+		matches.append(findAllMatches(matcher, "", root ? root : manager->root()));
```

Comment:
A node can only ever belong to one manager. Thus if `root` is provided, you should not loop over all managers.